TestPlans/IPadTests.xctestplan

```diff
@@ -18,45 +18,9 @@
   },
   "testTargets" : [
     {
-      "skippedTests" : [
-        "AssignmentDetailsTests",
-        "AssignmentsTests",
-        "CalendarTests",
-        "DashboardEmptyTests",
-        "LoginFindSchoolTests",
-        "LoginStartTests",
-        "MockedInboxTests",
-        "ModulesTests",
-        "Notifications2ListTests",
-        "NotificationsListTests",
-        "PeopleTests",
-        "QuizTests",
-        "StudentAccountNotificationsTests",
-        "StudentActAsUserTests",
-        "StudentAnnouncementTests",
-        "StudentCourseFileTests",
-        "StudentCourseGroupTests",
-        "StudentDashboardTests",
-        "StudentDeepLinkTests",
-        "StudentDiscussionDetailsTests",
-        "StudentDiscussionEditTests",
-        "StudentDiscussionListTests",
-        "StudentDiscussionReplyTests",
-        "StudentInboxTests",
-        "StudentLoginTests",
-        "StudentMockedInboxTests",
-        "StudentPageTests",
-        "StudentProfileTests",
-        "StudentTodoListTests",
-        "StudentUITestCase",
-        "StudentUserFilesTests",
-        "SubmissionButtonTests",
-        "SubmissionCommentsTests",
-        "SubmissionDetailsTests",
-        "SubmissionFilesTests",
-        "SyllabusTests",
-        "TextSubmissionTests",
-        "URLSubmissionTests"
+      "selectedTests" : [
+        "IPadAssignmentsTest\/testAssignments()",
+        "StudentSpringBoardTests\/testMultitaskingSetup()"
```

Comment:
Oh this is much better. Were you able to make Xcode do this or did you have to edit the test plan file directly?

Reply:
It was just two checkboxes named "automatically include new tests"